src/store/actions/settings.test.js

```diff
@@ -0,0 +1,54 @@
+import { actionTypes } from '@constants';
+import {
+  settingsReset,
+  settingsUpdated,
+  settingsRetrieved,
+} from './settings';
+import { getFromStorage } from '../../utils/localJSONStorage';
+
+describe('actions: setting', () => {
+  const dispatch = jest.fn();
+  const settings = { autoLog: false };
+
+  afterEach(() => {
```

Comment:
this should be at global level jest restore

Reply:
Done